### DISCUSSION TITLE
Add user intuitive way to turn auto numbering on or off in the Workbench

### DISCUSSION THREAD

**Discussion by tlammer** [specify/specify7#5557] [Feature Requests]
The current way to turn on auto numbering is to enter a placeholder value in the catalog number field, which is not intuitive for users.
Reported by Cryoarks

**Comment by grantfitzsimmons** [comment]
Still something that confuses new users. Here's how  you can use auto numbering in the WorkBench if you're curious:

> <p>For example, if your catalog number is formatted as a 9-digit number you need to insert <code>#########</code></p>
> <p>If you are using one of these example formatters, you would enter the following in the cell:</p>
> <div class="md-table fullscreen-table-wrapper" data-table-index="0">
> <div class="fullscreen-table-wrapper--buttons"></div></div>
> 
> Formatter | Cell Value
> -- | --
> AccessionNumber | 2023-AA-###
> AccessionNumberByYear | 2023-AA-AAA
> AccessionStringFormatter | AAAAAAAAAA
> CatalogNumber | 2023-######
> CatalogNumberAlphaNumByYear | 2023-######
> CatalogNumberNumeric | #########
> DeaccessionNumber | 2023-AA-###
> GiftNumber | 2023-###
> InfoRequestNumber | 2023-###
> LoanNumber | 2023-###
> NumericBigDecimal | ###############
> NumericByte | ###
> NumericDouble | ##########
> NumericFloat | ##########
> NumericInteger | ##########
> NumericLong | ##########
> NumericShort | #####
